packages/combobox/src/ComboboxOption.tsx

```diff
@@ -27,48 +29,71 @@ const comboboxOptionStyle = css`
   cursor: pointer;
   overflow: hidden;
   font-family: ${fontFamilies.legacy};
-  font-size: var(--lg-combobox-item-font-size);
-  line-height: var(--lg-combobox-item-line-height);
-  padding: var(--lg-combobox-item-padding-y) var(--lg-combobox-item-padding-x);
 
   // Left wedge
   &:before {
     content: '';
     position: absolute;
     left: 0;
     width: 3px;
-    height: var(--lg-combobox-item-wedge-height);
+    height: 22px;
     background-color: transparent;
     border-radius: 0 2px 2px 0;
     transform: scaleY(0.3);
```

Comment:
Like with transform, I believe the best practice here would be to use `scale3d` instead of `scaleY` to take advantage of hardware accelerated transitions.